client/api_off_chain_auth.go

```diff
@@ -79,6 +88,20 @@ Resources:
 - SP %s (name: SP_001) with nonce: %s`
 )
 
+// RegisterEDDSAPublicKey register EdDSA public key of this client for the given sp address and spEndpoint
```

Comment:
RegisterEDDSAPublicKey - Register EdDSA public key of this client for the given sp address and spEndpoint.